### PR TITLE
Add more detail to comments, please review.

### PR DESCRIPTION
**Pull request by Lonniebiz**
Some of my changes are helpful, but please review to make sure they are 100% correct.

### CODE REVIEW

File: esm/hyper/wire.js [hunk 1]
```diff
@@ -10,13 +10,13 @@ import render from './render.js';
 // all wires used per each context
 const wires = new WeakMap;
 
-// A wire is a callback used as tag function
+// A wire is a callback used as tag a function
```
Comment (WebReflection) [inline esm/hyper/wire.js]: the `tag function` is a concept itself described also in MDN as such

**Comment by Lonniebiz** [comment]
Sorry, I don't have much experience with github. Perhaps, I tried to edit too much at once.

Most importantly, I thought my edits to the comment on fragments made the comment more explicit and quicker to fully understand.

**Comment by WebReflection** [comment]
> I thought my edits to the comment on fragments made the comment more explicit and quicker to fully understand.

all but one, because `tag function` is a specific term, while _tag a function_ means nothing in that context.

File: esm/hyper/wire.js (hunk 1 above)
Comment (Lonniebiz) [reply esm/hyper/wire.js]: I agree with you, but I'm trying to figure out how to remove that "a" so it is back to the way it was.

**Comment by WebReflection** [comment]
Look, I've manually pushed those other two so it should be good now. Thanks anyway!

File: esm/hyper/wire.js (hunk 1 above)
Comment (Lonniebiz) [reply esm/hyper/wire.js]: I couldn't figure out how to edit this in the review stage to take out the "a". Instead I submitted another edit of just the main comment I was trying to enhance. Please have patients with me. I will be more helpful once I learned the fundamentals of github.

**Pull request closed without merging**